core/services/chainlink/application.go

```diff
@@ -677,6 +677,19 @@ func (app *ChainlinkApplication) RunJobV2(
 			CreatedAt:      t,
 			FinishedAt:     &t,
 		}, nil, false)
+		elapsed := time.Since(t)
+
+		// For testing metrics.
+		pipeline.PromPipelineTaskExecutionTime.WithLabelValues(fmt.Sprintf("%d", jb.ID), jb.Name.String, jb.Type.String()).Set(float64(elapsed))
+		var status string
+		if err != nil {
+			status = "error"
+			pipeline.PromPipelineRunErrors.WithLabelValues(fmt.Sprintf("%d", jb.ID), jb.Name.String).Inc()
```

Comment:
What about just having a counter for completed/error and setting a label for status?

Reply:
This is just the same as the existing pattern which we've already shipped - I don't think its worth the churn for nops